internal/handler/msg_saslcontinue.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handler
+
+import (
+	"context"
+
+	"github.com/FerretDB/FerretDB/internal/clientconn/conninfo"
+	"github.com/FerretDB/FerretDB/internal/handler/common"
+	"github.com/FerretDB/FerretDB/internal/handler/handlererrors"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// MsgSASLContinue implements `saslContinue` command.
+func (h *Handler) MsgSASLContinue(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
+	doc, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	var payload []byte
+
+	// most drivers follow spec and send payload as a binary
```

Comment:
This comment suggests there was a driver that sends payload in a different format (string?), do we want to keep allowing that?

Reply:
Discussed this with @b1ron. He checked that no driver seems to be doing this for SCRAM and we decided to remove this comment for SCRAM. The alternative would be to [actually] add code to handle that:

```go
stringPayload, err := common.GetRequiredParam[string](doc, "payload")
if err == nil {
	if payload, err = base64.StdEncoding.DecodeString(stringPayload); err != nil {
		return "", "", handlererrors.NewCommandErrorMsgWithArgument(
			handlererrors.ErrBadValue,
			fmt.Sprintf("Invalid payload: %v", err),
			"payload",
		)
	}
}
```

Reply:
Let's not add code. It's good news if all driver use binary :+1: